module3/labs/Week4/DeleteResource.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Delete Resource Lab
+---
+
+## Setup
+
+You've already had a chance to practice implementing Delete by building a delete button for reviews, so for today's lab we're going to do something a bit different than what we covered in class. 
+
+You're going to continue working on the MVC movies app and implement new features to make the application more user-friendly.
+
+## Brainstorm Features
+
+In software development, a `feature` is a specific piece of functionality that fulfills a specific user need or requirement. 

```

Comment:
I'm torn about if this is a good place to introduce full user stories with the full structure

“As a [persona], I [want to], [so that].” 

I don't want to pack in too much new stuff into a day, but also it might fit well here? Not sure.

Reply:
Thinking about this more, I don't think this is really the right place to introduce user stories, because what we're asking for here isn't really user stories. It's the actual specific features to implement, not what the user wants to be able to do.